src/lib/mapping/sign.ts

```diff
@@ -1,19 +1,27 @@
 import { hash } from "../utils/arweaveUtils";
 import { bufferTob64Url } from "../utils/arweaveUtils";
 import { signature as getSignedData } from "./signature";
-import { getActivePublicKey } from "./getActivePublicKey";
+import type Transaction from "arweave/web/lib/transaction";
+import { getCachedUserPublicKey } from "../auth/auth0";
+
+// TODO: Rename to signTransaction
 
 /**
  * Sign the given transaction. This function assumes (and requires) a user is logged in and a valid arweave transaction.
  * @param transaction The transaction to sign.
  * @returns The signed version of the transaction.
  */
-export async function sign(transaction: any, analyticsTags?: { name: string; value: string }[]): Promise<any> {
-  const owner = await getActivePublicKey();
+export async function sign(
+  transaction: Transaction,
+  analyticsTags?: { name: string; value: string }[],
```

Comment:
The intention here might have been to hardcode / add some othent tags to keep track of apps using othent.